config/locales/en.json

```diff
@@ -556,5 +562,17 @@
 	"ma.view.label.type_of_work": "Type of work:",
 	"ma.view.label.type_of_work_detail": "Type of work detail:",
 	"ma.view.label.functional_limitations": "Functional limitations:",
-	"ma.view.label.future_treatments": "Future treatments/impact on functional abilities:"
+	"ma.view.label.future_treatments": "Future treatments/impact on functional abilities:",
+	"Email is required": "Email is required",
+	"Does not seem to be a valid email": "Does not seem to be a valid email",
+	"City is required": "City is required",
+	"Province is required": "Province is required",
+	"Country is required": "Country is required",
+	"Postal code is required": "Postal code is required",
+	"Birthdate birthdate": "Birthdate birthdate",
+	"Birthdate is not a valid date": "Birthdate is not a valid date",
+	"Birthdate is not formatted correctly": "Birthdate is not formatted correctly",
```

Comment:
If the user prefers, are they able to use month names, then we convert them to a number?

Reply:
Not if we need to store the date as structured data (ie, an actual date-type in the database). Can't rely on user-inputted text for this.

Reply:
Okie dokie. Can the validation allow single digits, so '1' instead of '01', for example? Other than that, good to go.

Reply:
ah yes, can do that, and done that!